00-basics/002-javascript/Kristina/Learning002.js

```diff
@@ -0,0 +1,143 @@
+let RobotDinosaurFromSpace = {};
+RobotDinosaurFromSpace.roar = function(line) {
+    console.log(`The creature roars '${line}'`);
+};
+
+RobotDinosaurFromSpace.roar("I shall eat all of you puny humans ");
+function roar (line) {
+    console.log(`The ${this.type} Robot Dinosaur From Space Roars '${line}'`);
```

Comment:
This doesn't work properly, right? `this` is only accessible within classes, and the way you assign it to plain object shouldn't provide it with access to the object, unless you do explicit `bind` or `call`.